### DISCUSSION TITLE
Write exif for favorites [external library]

### DISCUSSION THREAD

**Discussion by cunicode** [immich-app/immich#8376] [Feature Request]
Related to being able to :rw to external (https://github.com/immich-app/immich/discussions/5449)

When I favorite an image, it could "rate 5" in the exif, which will carry the "favoriteness" across softwares.

My usecase: I use xnView to sort/rename/browse/edit/favorite my images locally.
And immich to explore on-the-go (on the sofa). 
If I favorite (or delete) a photo with Immich I'd like the changes to be reflected at file level.

**Comment by mmomjian** [comment]
Duplicate of #1696 and #3619

**Comment by jmathai** [comment]
The issues this was marked a duplicate of do not store the changes in the asset file's EXIF.

I created a containerized service which can be added to `docker-compose.yaml` which stores favorites, geolocation, description and albums to the EXIF of the asset itself. It is meant to work with external libraries.

https://github.com/jmathai/immich-exif
